telos-backend/src/routes/api/journal.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+
+const router = express.Router();
+
+router.get('/', (req, res) => {
```

Comment:
The endpoints for journal are not as self-explanatory as the individual widgets - could you quickly explain what each endpoint does, perhaps as a quick comment above each one?

Reply:
Done!